jina/clients/base/websocket.py

```diff
@@ -127,6 +134,8 @@ def _request_handler(request: 'Request') -> 'asyncio.Future':
                         p_bar.update()
                     yield response
 
+                await receive_task
```

Comment:
I hope that this little innocent line doesn't break some async magic, a second pair of eyes would be nice!

Reply:
why is this needed?

Reply:
otherwise the task is never awaited. Usually this is not a problem, since it essentially just performs a background task, but if it throws an Exception (which now it does), the Exception needs to be retrieved by `await`ing.